internal/workos/workos.go

```diff
@@ -2,5 +2,5 @@ package workos
 
 const (
 	// Version represents the SDK version number.
-	Version = "v0.5.0"
+	Version = "v0.6.0"
```

Comment:
This isn't a breaking change so I think we can just bump to 0.5.1

Reply:
@maxchehab @rjvani I think this would be minor bump based on our [versioning policy](https://github.com/workos-inc/workos/wiki/SDK-Release-Process#versioning-policy) since this isn’t a bug fix.